application/src/main/java/org/togetherjava/tjbot/commands/help/AskCommand.java

```diff
@@ -133,11 +137,28 @@ private boolean handleIsValidTitle(@NotNull CharSequence title, @NotNull IReplyC
     private @NotNull RestAction<Message> handleEvent(@NotNull InteractionHook eventHook,
             @NotNull ThreadChannel threadChannel, @NotNull Member author, @NotNull String title,
             @NotNull String category, @NotNull Guild guild) {
+        mapUserToDatabase(author, threadChannel);
         return sendInitialMessage(guild, threadChannel, author, title, category)
             .flatMap(any -> notifyUser(eventHook, threadChannel))
             .flatMap(any -> helper.sendExplanationMessage(threadChannel));
     }
 
+    private void mapUserToDatabase(Member author, ThreadChannel threadChannel) {
```

Comment:
the naming on this method is confusing. it is totally missing the context. instead, id go for `writeHelpThreadToDatabase` or similar